src/go/cmd/update-collection-v3/valuesv2.go

```diff
@@ -0,0 +1,1804 @@
+package main
+
+type ValuesV2 struct {
+	NameOverride     *string `yaml:"nameOverride,omitempty"`
+	FullnameOverride *string `yaml:"fullnameOverride,omitempty"`
+	Sumologic        struct {
+		SetupEnabled   *bool   `yaml:"setupEnabled,omitempty"`
+		CleanupEnabled *bool   `yaml:"cleanupEnabled,omitempty"`
+		Endpoint       *string `yaml:"endpoint,omitempty"`
+		HTTPProxy      *string `yaml:"httpProxy,omitempty"`
+		HTTPSProxy     *string `yaml:"httpsProxy,omitempty"`
+		NoProxy        *string `yaml:"noProxy,omitempty"`
+		ClusterName    *string `yaml:"clusterName,omitempty"`
+		Cluster        struct {
+			Host                 *string `yaml:"host,omitempty"`
+			ClusterCaCertificate *string `yaml:"cluster_ca_certificate,omitempty"`
+			Token                *string `yaml:"token,omitempty"`
+			LoadConfigFile       *bool   `yaml:"load_config_file,omitempty"`
+		} `yaml:"cluster,omitempty"`
+		CollectionMonitoring *bool    `yaml:"collectionMonitoring,omitempty"`
+		PodLabels            struct{} `yaml:"podLabels,omitempty"`
```

Comment:
does it deserialize:

```
podLabels:
  label1: value1
```
correctly?

Reply:
Good point. This will probably have to be either defined as struct or as `map[string]interface{}`.

Given this is just a boilerplate PR I'll leave this for future PRs to decide and add tests for it :)

Reply:
TBH I am not sure now if typed language is good for the migration case
On the other hand we will have a typed schema for our collection